backend/assignments/1/src/main/java/shrishti/example/Logger/logger.java

```diff
@@ -0,0 +1,18 @@
+package shrishti.example.Logger;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+public class logger {
+    private static final Logger logger
+            = LoggerFactory.getLogger(logger.class);
+
+
+    public static void printOutput(String message) {
```

Comment:
Making a logger sub-module. Exposing generic methods for logger rather than calling the instances is cleaner and re-usable among projects.